unity/Assets/Scripts/UnityMonoBehaviour/UnityUI.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class UnityUI : MonoBehaviour
+{
+    public void exitGame()
```

Comment:
rename function as this is ambiguous to exiting the entire "game", as in not to the actual round

Reply:
fixed in 10d35704b2bdaa410737e6ed888afb3ba8b3769a